### ISSUE TITLE
Numbers in secrets name

### ISSUE THREAD

**Issue by 12nick12** [metroline/metroline#31]
I was trying to create a secret called BASE64_SSH_KEY and it wasn't able to work. I tested the name on an online regex checker and it passed.

**Comment by gempain** [comment]
That's strange, it should work as the regex used is [this one](https://github.com/metroline/metroline/blob/1e8a1d8f1681b99e471c2adcd5b3a27ef3bbf24d/src/server/constants.ts#L11):

```
export const ENV_VAR_NAME_PATTERN = /^[a-zA-Z_][a-zA-Z0-9_]*$/;
```

**Comment by 12nick12** [comment]
I'm not getting any errors, the pipeline just isn't registering the secret. It's acting like the secret is just the secrets name. As soon as I remove the 64 from the name it works.

**Comment by 12nick12** [comment]
This secret would be use as '{{BASE64_SSH_KEY}}' right?

**Comment by 12nick12** [comment]
OK, I also tried to map the secret to an env variable and then echo that varaible `echo "${VARIABLE_NAME}"` and it still showed up as '{{BASE64_SSH_KEY}}'

I didn't think to try it as a global variable though.

**Comment by 12nick12** [comment]
Thank you guys. Other than that I'm loving it. Much better/simpler than DroneCI.

**Comment by gempain** [comment]
@12nick12 I can confirm this is a bug. I'm on it.

**Comment by gempain** [comment]
Released in metroline-server 1.3.1

**Comment by gempain** [comment]
Very welcome, and thank YOU for all the kind words. This is what makes open source awesome.
